synapse_data_warehouse/synapse_raw/tables/V2.32.2__refresh_nodesnapshots.sql

```diff
@@ -0,0 +1,75 @@
+-- Configure environment
+USE SCHEMA {{database_name}}.synapse_raw; --noqa: JJ01,PRS,TMP,CP02
+USE WAREHOUSE compute_medium;
+
+SET cutoff_date = '2025-01-22';
+SET num_records_to_be_reloaded = (
+    SELECT count(*)
+    FROM nodesnapshots
+    WHERE snapshot_date >= $cutoff_date
+);
+
+-- This scripting block is an attempt to avoid any `COPY INTO` action in
+-- the unlikely case of a rollback/replay of our database state. For example,
+-- if we are deploying our database from scratch -- or more specifically, if
+-- we don't have any table data requiring a `version_history` backfill -- we don't
+-- want to load data (that's a job for the associated data loading task).
+BEGIN 
```

Comment:
It appears that you used a previous PR of mine as a template, although using a scripting block was just my original solution. I can't remember exactly why, but ultimately I removed the scripting block (I think maybe schemachange had some issues?).

You can find the backfill script that ultimately got merged [here](https://github.com/Sage-Bionetworks/snowflake/blob/main/synapse_data_warehouse/synapse_raw/tables/V2.25.2__refresh_node_snapshot.sql).